### PR TITLE
Update readme with the required environment variables

### PR DESCRIPTION
**Pull request by ke4**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,18 +33,17 @@ ignored by the version control system.
 
 ##### Running a Single Test
 
-If you haven't already done so, you will need to configure the HCA CLI tool to use the EBI installation of the upload service rather than the DCP (default) version.
+To run a single test, make sure that all necessary environment variables are provided.
 
 ```
-./setup_ingest_config.sh
```
Comment (amnonkhen) [inline README.md:42]: We should also delete the file from git, shouldn't we?

File: README.md [hunk 2]
```diff
@@ -36,18 +33,17 @@ ignored by the version control system.
...
-```
-
-To run a single test, make sure that all necessary environment variables are provided. At the time of writing, the most
-commonly required variables are `DEPLOYMENT_ENV` and `GOOGLE_APPLICATION_CREDENTIALS`.
-
-```
-export AWS_PROFILE=embl-ebi \
+export AWS_PROFILE=embl-ebi; \
```
Comment (amnonkhen) [inline README.md:39]: I think we should start using `.env` files.

File: README.md [hunk 3]
```diff
@@ -36,18 +33,17 @@ ignored by the version control system.
 
 ##### Running a Single Test
 
-If you haven't already done so, you will need to configure the HCA CLI tool to use the EBI installation of the upload service rather than the DCP (default) version.
+To run a single test, make sure that all necessary environment variables are provided.
```
Comment (amnonkhen) [inline README.md:36]: I tried to setup using the instructions here and got an error about missing 'client_email'.

**Review by amnonkhen — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ke4) [reply README.md:42]: yep

File: README.md (hunk 2 above)
Comment (ke4) [reply README.md:39]: maybe.
@amnonkhen Could you create a ticket for that one, please?

File: README.md (hunk 3 above)
Comment (ke4) [reply README.md:36]: Could we take this offline, please?
As it is perfectly working for all of us, but you.
Tx

File: README.md (hunk 1 above)
Comment (ke4) [reply README.md:42]: done

File: README.md [hunk 4]
```diff
@@ -36,18 +33,17 @@ ignored by the version control system.
...
-```
-export AWS_PROFILE=embl-ebi \
+export AWS_PROFILE=embl-ebi; \
 export DEPLOYMENT_ENV=dev; \
+export INGEST_API_JWT_AUDIENCE=https://dev.data.humancellatlas.org/; \
+export DEPLOYMENT_STAGE=${DEPLOYMENT_ENV}; \
+export ARCHIVER_API_KEY=$(aws secretsmanager get-secret-value --region us-east-1 --secret-id ingest/${DEPLOYMENT_ENV}/secrets --query SecretString --output text | jq -jr '.archiver_api_key'); \
+export HCA_UTIL_ADMIN_ACCESS=$(aws secretsmanager get-secret-value --region us-east-1 --secret-id hca/util/aws-access-keys --query SecretString --output text | jq -jr '.ADMIN_ACCESS_KEY'); \
+export HCA_UTIL_ADMIN_SECRET=$(aws secretsmanager get-secret-value --region us-east-1 --secret-id hca/util/aws-access-keys --query SecretString --output text | jq -jr '.ADMIN_SECRET_ACCESS_KEY'); \
+export HCA_UTIL_ADMIN_PROFILE='test-hca-util-admin'; \
```
Comment (amnonkhen) [inline README.md:46]: Is the existence of this profile in the aws configuration a prerequisite to running the tests?

**Review by amnonkhen — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (aaclan-ebi) [reply README.md:46]: No, it's not. The profile is being set up thru hca-util and is being done in the test. Please see https://github.com/ebi-ait/ingest-integration-tests/blob/dev/tests/runners/submission_manager.py#L40-L44

**Review by amnonkhen — APPROVED** [review]

**Review by aaclan-ebi — APPROVED** [review]

**Pull request merged**